pkg/openfeature/provider.go

```diff
@@ -12,15 +12,17 @@ const (
 	TargetingKey string = "targetingKey" // evaluation context map key. The targeting key uniquely identifies the subject (end-user, or client service) of a flag evaluation.
 )
 
+type TransformedContext map[string]interface{}
```

Comment:
Consider adding a comment explaining how the targeting key is stored within this map

Reply:
Nitpick:

`transformedContext` is used in a lot of SDKs as a variable name internally because it was the context provided after the `contextTransformer` function ran.

I'm fine with it here, but I wonder if `FlattenedContext` might be a bit more helpful.